.gitignore

```diff
@@ -142,6 +142,9 @@ dmypy.json
 modules/
 *.swp
 
+# VsCode
+.vscode
```

Comment:
do we need to change this

Reply:
I'm happy to remove it if you would prefer it not be part of this PR. I just kept accidentally adding my vscode files and added it to prevent that from happening again but understand if you want a separate PR.

Reply:
eh I think it's fine. People shouldn't be committing changes to the `.vscode` file anyways